src_ext/Makefile

```diff
@@ -84,6 +87,16 @@ distclean:
 
 LIB_EXTS = extlib re cmdliner graph cudf dose_common dose_debian dose_algo uutf jsonm
 
+# add the preprocessor directive for OCamlMakefile
+CPPO = cppo/cppo
+addcppo = for ml in $1/*.ml $1/*.mli; do echo '(*pp $(CPPO) *)' | cat - $$ml > temp && mv temp $$ml; done
+
+addcppo:
+	$(MAKE) -C cppo all
+	$(call addcppo,dose/common)
```

Comment:
Now we use sed instead of cat. I don't need we need a timestamp as the target clone is always run before addcppo.